pkg/query/querier.go

```diff
@@ -210,18 +210,20 @@ func newQuerierWithOpts(
 
 	partialResponseStrategy := storepb.PartialResponseStrategy_ABORT
 	if opts.GroupReplicaPartialResponseStrategy {
-		level.Debug(logger).Log("msg", "Enabled group-replica partial response strategy in newQuerierInternal")
+		level.Info(logger).Log("msg", "Enabled group-replica partial response strategy in newQuerierInternal")
```

Comment:
This will be very chatty. I intentionally changed it to Debug previously.

Reply:
Every query will print this log.

Reply:
found i actually doesn't need this log line, it is logged here already: ```
	level.Info(logger).Log("msg", "databricks querier features", "opts", fmt.Sprintf("%+v", opts))
```